src/routing_table/authority.rs

```diff
@@ -20,6 +20,7 @@ use std::fmt::{self, Binary, Debug, Display, Formatter};
 /// `PrefixSection` use _section_ verification: the set from which a quorum is required is all
 /// members of the section (`Section`) or of all sections matching the prefix (`PrefixSection`).
 #[derive(Serialize, Deserialize, PartialEq, PartialOrd, Eq, Ord, Clone, Copy, Hash)]
+#[allow(clippy::large_enum_variant)]
```

Comment:
What changed, is it the PublicId now bigger?

Reply:
Hm, I just checked. In fleming, `PublicId` is 97 bytes, in this PR it's 96. So it's actually smaller. Then I tried to run clippy on fleming and that lint got triggered also there. No idea how that get through CI :thinking: